docs/error_knowledge_base.md

```diff
@@ -418,14 +419,36 @@ See also: [Submitting a Package](how_to_add_packages.md#submitting-a-package).
 The recipe missess [short_paths](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths) attribute.
 It may happen due to the very long paths within source, build or package directories during the package creating.
 Consider adding the following code:
+
 ```python
 class SomeRecipe(ConanFile):
     ...
 
     short_paths = True
 ```
 
-See also: (Maximum Path Length Limitation)[https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd].
+See also: [Maximum Path Length Limitation](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd).
+
+#### **<a name="KB-H068">#KB-H068</a>: "TEST_TYPE MANAGEMENT"**
+
+https://github.com/conan-io/hooks/commit/ad867ad0a3656fe77ecc964e9b8f3657995f0d5d
+
+#### **<a name="KB-H069">#KB-H069</a>: "TEST PACKAGE - NO DEFAULT OPTIONS"**
+
+https://github.com/conan-io/hooks/commit/7349d593d0728ae665dcb2c92ed3038b3cbb8ee5
+
+#### **<a name="KB-H070">#KB-H070</a>: "MANDATORY SETTINGS"**
+
+https://github.com/conan-io/hooks/commit/3e7e77fa5d14ee53dc1ad6b10d85358871fdd037
```

Comment:
**KB-H070**: We agreed to list always the settings and remove those that were not needed in `package_id` method. There is still some open discussion around listing always all the settings, or just the needed ones.

Reply:
an earlier discussion we had about it (or at least one of them): https://github.com/conan-io/conan-center-index/issues/3929

Reply:
and one more: https://github.com/conan-io/conan-center-index/issues/3482